core/src/main/scala/uk/gov/nationalarchives/tdr/api/core/graphql/GraphQlTypes.scala

```diff
@@ -32,6 +36,29 @@ object GraphQlTypes {
     }
   )
 
+  implicit private val localDateTimeDecoder: Decoder[LocalDateTime] = Decoder.decodeString.emap(str =>
+    Either.catchNonFatal(LocalDateTime.parse(str)).leftMap(t => "LocalDateTime")
+  )
+
+  private def parseDate(s: String) = Try(
+    LocalDateTime.ofInstant(Instant.ofEpochMilli(s.toLong), ZoneId.systemDefault())) match {
```

Comment:
Does this mean the API still parses dates as epoch timestamps?

I think it should accept ISO 8601 dates instead. It's the standard pretty much everywhere (including the [GDS API standards](https://www.gov.uk/guidance/gds-api-technical-and-data-standards#to-represent-time-and-date)), and it's also useful to have a human-readable format when you're inspecting API requests.

Reply:
Refactored so takes an ISO8601 date format, converts to java.Time.Instant and adds as a UTC timestamp in the DB.